server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -222,6 +222,7 @@ private DivTag createImageDescriptionForm(
                         .setPlaceholderText("Colorful fruits and vegetables in bins")
                         .setValue(form.value().get().getSummaryImageDescription())
                         .getInputTag()))
+        .with(p("Note: Image description is required before uploading an image.").withClass("mt-1"))
```

Comment:
nitpick: Can you also add `mb-1` to the class list? It looks a little close to the buttons under it.